rotkehlchen/tests/api/test_nfts.py

```diff
@@ -150,29 +152,92 @@ def test_nft_ids_are_unique(rotkehlchen_api_server):
 
 @requires_env([TestEnvironment.NIGHTLY, TestEnvironment.NFTS])
 @flaky(max_runs=3, min_passes=1)  # all opensea calls have become quite flaky
-@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4]])
+@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4, TEST_ACC5]])
 @pytest.mark.parametrize('start_with_valid_premium', [True])
 @pytest.mark.parametrize('ethereum_modules', [['nfts']])
 def test_nft_balances_and_prices(rotkehlchen_api_server):
     """Check that nfts balances return the expected fields. Also check nft prices"""
     response = requests.get(api_url_for(
         rotkehlchen_api_server,
         'nftsbalanceresource',
-    ), json={'async_query': False})
-
+    ), json={'async_query': False, 'ignore_cache': False})
     result = assert_proper_response_with_result(response)
-    for nfts_balances in result.values():
+    # check that nothing is returned with ignore_cache=False until nfts are queried
+    assert result['entries'] == {}
```

Comment:
comment here is confusing as to where the check is. I thought it was the next query. When you want to add human readable text to an assert you do this.

```suggestion
    assert result['entries'] == {}, ' nothing should e returned with ignore_cache=False until nfts are queried'
```